playbooks/byo/openshift-prometheus.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: generate prometheus deployment
+  hosts: masters
+  roles:
+  - openshift_prometheus
+  vars:
+    openshift_prometheus_state: present
+    # support kubernetes < 1.7
+    openshift_prometheus_use_default_metrics_path: True
```

Comment:
@joelddiaz this flag is for the default endpoint (before kubernetes 1.7).
remove it if you update to a newer cluster.